web/src/components/gnd-feature-type-editor/index.js

```diff
@@ -73,14 +73,16 @@ const styles = (theme) => ({
   },
   colorPickerTab: {
     position: 'absolute',
-    left: 145,
+    left: 165,
```

Comment:
Move this leftward since you have moved the default style, I tried and 142 looks like a good fit for you new change.

Reply:
Sgtm.